README.md

```diff
@@ -217,9 +217,8 @@ OpenJS Foundation Collaboration spaces are able to take advantage of several ser
 
 ### Help with project infrastructure
 
-In almost all situations, the best way to get support is to email
-[operations@openjsf.org](mailto:operations@openjsf.org) with your request. For
-example:
+In almost all situations, the best way to get support is to 
+[file a ticket with The Linux Foundation](https://members.openjsf.org). For example:
```

Comment:
It is concerning to me how disconnected this is from Foundation staff :/

Reply:
If it helps, it all goes to the same person eventually. We've just been asked to adjust our processes so they're less isolated to single person's email.